vrs_anvil/__init__.py

```diff
@@ -356,19 +358,24 @@ def check_paths(self) -> "Manifest":
         return self
 
 
-def query_metakb(id, log=False):
+def query_metakb(vrs_id, log=False):
```

Comment:
I think it could stay in since it's agnostic to analysis and data will be pulled in anyways. If we want to add CAF creation to this repo in the future  this will also be needed